modules/programs/eza.nix

```
@@ -29,8 +29,16 @@ with lib;
       default = true;
     };
 
-    enableFishIntegration = mkEnableOption "Fish integration" // {
-      default = true;
+    enableFishIntegration = mkOption {
+      type = types.enum [ "no" "aliases" "abbreviations" ];
```

Can keep backwards compatibility

```suggestion
      type = with types; either bool (enum [ "no" "aliases" "abbreviations" ]);
      apply = v: if isBool v then if v then "aliases" else "no" else v;
```

---

Change is now backwards-compatible.